tests/pytests/functional/modules/test_aptpkg.py

```diff
@@ -98,6 +98,38 @@ def revert_repo_file(tmp_path):
         aptpkg.refresh_db()
 
 
+@pytest.fixture
+def build_repo_file():
```

Comment:
Let's add the destructive test and skip of not root outlet markers since this changes the host file system.

Reply:
to the pytest fixture? didn't think that would do anything. the only test that uses it so far does have the destructive test marker. will add the skip if not root.

Reply:
Sorry, no, to the test.
And, if the test has those, disregard my comment.

Reply:
@whytewolf Looks like these markers where added to the fixture, was this intended?

Reply:
I misled @whytewolf.

They should be added to the tests using the fixture.

Reply:
i thought i removed those. let me check if that push went through